service/src/main/java/org/cbioportal/genome_nexus/service/internal/MyVariantInfoServiceImpl.java

```diff
@@ -65,7 +65,9 @@ public MyVariantInfo getMyVariantInfoByHgvsVariant(String variant)
             myVariantInfos = this.getMyVariantInfoByMyVariantInfoVariant(queryVariants);
             // manually set the original hgvs variant field
             for (MyVariantInfo myVariantInfo: myVariantInfos) {
-                myVariantInfo.setHgvs(queryToVariant.get(myVariantInfo.getQuery()));
+                // myVariantInfo.getVariant() will return my_variant_info query id (e.g. chr7:g.140453136A>T)
+                // queryToVariant.get() will return genome nexus query id (e.g. 7:g.140453136A>T)
+                myVariantInfo.setHgvs(queryToVariant.get(myVariantInfo.getVariant()));
```

Comment:
as discussed it might make sense to wrap this line in try/accept such that not entire loop stops when one of these fails